packages/app/src/pages/endpoints/[endpointId]/_/content/types/table/index.tsx

```diff
@@ -104,6 +112,18 @@ const ContentTable: React.FC<Props> = ({
     [document, base, sorts]
   );
 
+  const [rowObject, setRowObject] = useState<Data>();
+  const onRowClick = useCallback(
+    (data: Data) => {
+      drawer.open();
+      setRowObject(data);
+    },
+    [drawer]
+  );
+  const closeDrawer = () => {
+    drawer.close();
+  };
```

Comment:
fyi It's possible to pass drawer.close to the component like
close={drawer.close}

Reply:
This function is not necessary so I will adopt your suggestion.